apm-sniffer/apm-sdk-plugin/mysql-common/src/main/java/org/apache/skywalking/apm/plugin/jdbc/mysql/PreparedStatementExecuteMethodsInterceptor.java

```diff
@@ -61,7 +89,9 @@ public final Object afterMethod(EnhancedInstance objInst, Method method, Object[
         Class<?>[] argumentsTypes,
         Object ret) throws Throwable {
         StatementEnhanceInfos cacheObject = (StatementEnhanceInfos)objInst.getSkyWalkingDynamicField();
-        if (cacheObject.getConnectionInfo() != null) {
+        ConnectionInfo connectionInfo = cacheObject.getConnectionInfo();
+        String methodName = method.getName();
+        if (connectionInfo != null && !PS_SETTERS.contains(methodName) && !PS_IGNORED_SETTERS.contains(methodName)) {
```

Comment:
You should use tracing parameter config to judge first, for performance when it is off.